client/cl_ui.lua

```diff
@@ -287,14 +287,24 @@ function InitiateMenus(isMotorcycle, vehicleHealth, categories, welcomeLabel)
                     tempNum = tempNum + 1
 
                     if maxVehiclePerformanceUpgrades == 0 then
-                        populateMenu(v.category:gsub("%s+", "") .. "Menu", n.id, n.name, "$" .. vehicleCustomisationPrices.performance.prices[tempNum])
+
+                        if Config.UsePercentage then
+                            populateMenu(v.category:gsub("%s+", "") .. "Menu", n.id, n.name, "$" .. math.floor((vehiclePrice / 100) * vehicleCustomisationPricesPercentage.performance[tempNum]))
+                        else
+                            populateMenu(v.category:gsub("%s+", "") .. "Menu", n.id, n.name, "$" .. vehicleCustomisationPrices.performance.prices[tempNum])
+                        end
```

Comment:
```suggestion
local price = Config.UsePercentage and math.floor((vehiclePrice / 100) * vehicleCustomisationPricesPercentage.performance[tempNum]) or vehicleCustomisationPrices.performance.prices[tempNum]
populateMenu(v.category:gsub("%s+", "") .. "Menu", n.id, n.name, "$" ..price )
```
If you could make this change to all the entries to make it "cleaner" looking.